<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员界面</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>返回开始菜单</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>图书列表</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>110</number>
         </attribute>
         <column>
          <property name="text">
           <string>图书id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>书名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>作者</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>库存</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>位置</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>图书ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_bid">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>书名</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_bname"/>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>作者</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_bauthor"/>
             </item>
             <item>
              <widget class="QPushButton" name="Button_book_cr">
               <property name="text">
                <string>插入</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Button_book_sc">
               <property name="text">
                <string>删除</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>类型</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_btype"/>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>库存</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_bnumber"/>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>位置</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_bplace"/>
             </item>
             <item>
              <widget class="QPushButton" name="Button_book_gg">
               <property name="text">
                <string>更改</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Button_book_cz">
               <property name="text">
                <string>查找</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="Button_book_display">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>显示所有图书</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>借书列表</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableWidget" name="tableWidget_2">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>90</number>
         </attribute>
         <column>
          <property name="text">
           <string>序号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>书ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>书名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>用户账号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>用户名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>借阅时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>归还时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>是否归还</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>序号</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_rid"/>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>图书ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_rbid"/>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>书名</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_rbname"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>借书人账号</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_ruid"/>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>借书人姓名</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_runame"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>是否归还</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox"/>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>借阅日期在此之前:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_day">
               <property name="placeholderText">
                <string>yyyy-mm-dd</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="2">
            <widget class="QPushButton" name="Button_record_cz">
             <property name="text">
              <string>查找</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="Button_record_gg">
             <property name="text">
              <string>归还</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="Button_record_display">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>显示所有借书记录</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>用户列表</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTableWidget" name="tableWidget_3">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>账号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>姓名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>密码</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>性别</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>生日</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>用户账号</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_userid"/>
             </item>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>用户姓名</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_username"/>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>用户密码</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_userpassword"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>用户性别</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_2"/>
             </item>
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>用户生日</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_userbirthday">
               <property name="placeholderText">
                <string>yyyy-mm-dd</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QPushButton" name="Button_user_cr">
             <property name="text">
              <string>增加</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="Button_user_sc">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="Button_user_cz">
             <property name="text">
              <string>查找</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="Button_user_gg">
             <property name="text">
              <string>更改</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="Button_user_display">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>显示所有用户</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
